.github/actions/delete-deployments/action.yml

```diff
@@ -0,0 +1,56 @@
+name: delete-deployments
```

Comment:
This name should be more specific as the notion of "deployment" is vague: there are other actions handling deployment and its status, naming should be consistent with them see https://github.com/Alfresco/alfresco-build-tools?tab=readme-ov-file#github-deployment-create-and-github-deployment-status-update